.ci/hipify.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
I think this file should be added in a follow-up PR in which we'll enable hipifying at our CI or will request users hipify localy before suggesting a CUDA-related PRs.

Reply:
I agree. We can postpone this to the next PR for ROCm.

Reply:
Removed.